benchopt/base.py

```diff
@@ -183,24 +187,43 @@ def skip(self, **objective_dict):
 
         return False, None
 
+    def run_once(self):
```

Comment:
can we expose stop_val here and pass it L206 for more flexibility? Some solvers need to be run with more than 1 iteration, solvers based on tolerance could use another value too

Reply:
Good point